package/MDAnalysis/converters/ParmEd.py

```diff
@@ -171,21 +170,13 @@ def convert(self, obj):
         obj : AtomGroup or Universe or :class:`Timestep`
         """
         try:
-            # TODO: remove this guard when parmed has a release
-            # that supports NumPy 2
-            if NumpyVersion(np.__version__) < "2.0.0":
-                import parmed as pmd
-            else:
-                raise ImportError
+            import parmed as pmd
         except ImportError:
-            if NumpyVersion(np.__version__) >= "2.0.0":
-                ermsg = "ParmEd is not compatible with NumPy 2.0+"
-            else:
-                ermsg = (
-                    "ParmEd is required for ParmEdConverter but is not "
-                    "installed. Try installing it with \n"
-                    "pip install parmed"
-                )
+            ermsg = (
```

Comment:
```suggestion
            errmsg = (
```

Would be great to fix this pre-existing typo! (currently triggering in a user guide run here: https://github.com/MDAnalysis/UserGuide/actions/runs/13603154438/job/38031532193?pr=408)